Raywire/iReporter
# [Finish #163176704] Shows correct number of returned results on search

**What does this PR do?**
Shows correct number of returned results on search
**Descriptions of the tasks to be completed?**
Display the correct number of search results
**How should this be manually tested?**
Go to https://raywire.github.io/iReporter/UI/interventions.html or https://raywire.github.io/iReporter/UI/redflags.html
Search using any parameter and notice the number of returned results displayed at the bottom
**Pivotal tracker story**
#163176704
